web/client/components/maps/modals/MetadataModal.jsx

```diff
@@ -123,7 +123,7 @@ const MetadataModal = React.createClass({
     },
     componentDidUpdate(prevProps) {
         if (this.props.show && !prevProps.show) {
-            if (this.props.displayPermissionEditor) {
+            if (this.props.displayPermissionEditor && this.props.user.name === this.props.map.owner) {
```

Comment:
ADMIN role users should be able to edit map permissions even if they are not owners